src/Bicep.Core/TypeSystem/Radius/KnownTypes.cs

```diff
@@ -209,6 +209,27 @@ public static NamedObjectType MakeServiceBusQueue()
             return MakeComponentType(kind, config: configType, bindings: bindings);
         }
 
+        public static NamedObjectType MakeRedis()
+        {
+            var kind = "redislabs.com/Redis@v1alpha1";
+            var configType = new NamedObjectType(
+                "config",
+                validationFlags: TypeSymbolValidationFlags.WarnOnTypeMismatch,
```

Comment:
Also if you want to test, you can just F5 in vscode and it will run the extension for you.